resources/strings/strings.bak

```diff
@@ -0,0 +1,104 @@
+<strings>
```

Comment:
Please remove this file, as I think it's not longer needed.

Reply:
Ok, no problem